lib/Notes/NoteRenderer.js

```diff
@@ -125,34 +125,42 @@ class NoteRenderer extends React.Component {
                 open={this.state.dropDownOpen}
                 onToggle={this.onToggleDropDown}
               >
-                <Button
-                  buttonStyle="link slim"
-                  aria-label={formatMessage({ id: 'stripes-smart-components.editOrDeleteNote' })}
-                  bsRole="toggle"
-                >
-                  <Icon icon="down-caret" color="rgba(150, 150, 150, .5)" />
-                </Button>
-                <DropdownMenu
-                  bsRole="menu"
-                  width="5em"
-                  minWidth="5em"
-                  aria-label={formatMessage({ id: 'stripes-smart-components.editOrDeleteNote' })}
-                  onToggle={this.onToggleDropDown}
-                >
-                  <IfPermission perm="notes.item.put">
-                    <Button buttonStyle="marginBottom0 hover fullWidth" onClick={this.handleClickEdit}>
-                      <FormattedMessage id="stripes-components.button.edit" />
-                    </Button>
-                  </IfPermission>
-                  <IfPermission perm="notes.item.delete">
+                <FormattedMessage id="stripes-smart-components.editOrDeleteNote">
+                  {ariaLabel => (
                     <Button
-                      buttonStyle="marginBottom0 hover fullWidth"
-                      onClick={() => { this.handleClickDelete(noteKey); }}
+                      buttonStyle="link slim"
+                      aria-label={ariaLabel}
+                      bsRole="toggle"
                     >
-                      <FormattedMessage id="stripes-components.button.delete" />
+                      <Icon icon="down-caret" color="rgba(150, 150, 150, .5)" />
                     </Button>
-                  </IfPermission>
-                </DropdownMenu>
+                  )}
+                </FormattedMessage>
+                <FormattedMessage id="">
```

Comment:
I think since both of these messages are `editOrDeleteNote`, this `FormattedMessage` without an `id` probably isn't meant to exist.